src/utils/recommendation.ts

```diff
@@ -165,15 +165,15 @@ export const generateUser = ({
     Date.parse(
       `2022-11-01T${startHours.toString().padStart(2, "0")}:${startMinutes
         .toString()
-        .padStart(2, "0")}:00Z`
+        .padStart(2, "0")}:00-05:00`
     )
   );
   const [endHours, endMinutes] = endTime.split(":").map((s) => _.toInteger(s));
   const endDate = new Date(
     Date.parse(
       `2022-11-01T${endHours.toString().padStart(2, "0")}:${endMinutes
         .toString()
-        .padStart(2, "0")}:00Z`
+        .padStart(2, "0")}:00-05:00`
     )
   );
 
```

Comment:
Next Steps: 

- Make sure new users being created are having their times saved in EST (check in profile.tsx), and the trpc ("me") endpoint
- Correct tests